lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -314,18 +318,18 @@ extension ASTGenVisitor {
 // 'self.visit(<expr>)' recursion pattern between optional and non-optional inputs.
 extension ASTGenVisitor {
   @inline(__always)
-  func generate(_ node: TypeSyntax?) -> ASTNode? {
+  func generate(_ node: TypeSyntax?) -> BridgedTypeRepr {
     guard let node else {
-      return nil
+      return .init(raw: nil)
```

Comment:
1. I’m not a huge fan of `.init` because it’s usually unclear which type is being created. `BridgedTypeRepr(raw: nil)` is a little bit longer to write but a lot easier to read IMO.
2. Have you considered conforming `BridgedTypeRepr` etc. to `ExpressibleByNilLiteral`?

Reply:
I wanted to keep this fairly verbose since ASTGen shouldn't need to construct `nil` AST nodes all that often, and it should be made clear that it's burying the `nil` value in a non-optional type. I'm happy to change to `BridgedTypeRepr(raw: nil)`

Reply:
Sounds good to me. Changing to `BridgedTypeRepr(raw: nil)` would be appreciated.